material-ui-superselectfield/material-ui-superselectfield.d.ts

```diff
@@ -0,0 +1,45 @@
+declare module "material-ui-superselectfield" {
+    import * as React from "react"
+
+    interface IMaterialUISuperSelectFieldProps {
+        name: string;
```

Comment:
Only `name` and `value` appear to be required for the component, according to the [docs](https://github.com/Sharlaan/material-ui-superselectfield).